build.gradle.kts

```diff
@@ -115,6 +115,17 @@ tasks.withType<KotlinCompile>().configureEach {
     }
 }
 
+tasks.dokkaHtml {
```

Comment:
What puzzles me -- who (and when) is responsible for updating the doc and the site?